src/Stratosphere/Value.hs

```diff
@@ -100,33 +100,33 @@ mkFunc key args = JSON.object [(key, JSON.Array $ fromList args)]
 -- @List<AWS::EC2::Subnet::Id>@ then, you can use @RefList "SubnetIds"@ to
 -- reference it.
 data ValueList a
-  = ValueList [Value a]
-  | RefList Text
+  = GetAZs (Value Text)
   | ImportValueList (Value Text)
+  | RefList Text
   | Split Text (Value a)
-  | GetAZs (Value Text)
+  | ValueList [Value a]
   deriving (Show, Eq)
 
 instance IsList (ValueList a) where
   type Item (ValueList a) = Value a
   fromList = ValueList
 
   toList = \case
-    (ValueList xs) -> xs
-    -- This is obviously not meaningful, but the IsList instance is so useful
-    -- that I decided to allow it.
-    (RefList _)         -> []
+    (GetAZs _)          -> []
     (ImportValueList _) -> []
+    (RefList _)         -> []
     (Split _ _)         -> []
-    (GetAZs _)          -> []
+    -- This is obviously not meaningful, but the IsList instance is so useful
+    -- that I decided to allow it.
```

Comment:
__question:__ Is this comment in the correct location? Prior to this change it was *after* this node, and now it's before it. I wanted to make sure it was attributed to the correct place.